pytorch_lightning/trainer/supporters.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, window_length: int):
 
     def reset(self) -> None:
         """Empty the accumulator."""
-        self = TensorRunningAccum(self.window_length)
+        self.__init__(self.window_length)
```

Comment:
Could you add a test to make sure `TensorRunningAccum ` is having the right behaviour ?

Reply:
Sure. I've tested it manually, but I would add the test in the next commit as you suggested.

Reply:
I don't think we have tests for `TensorRunningAccum` at all, so you'll have to create them